lightning/src/routing/router.rs

```diff
@@ -4801,6 +4820,31 @@ mod tests {
 		assert_eq!(route.get_total_amount(), 0);
 	}
 
+	#[test]
+	fn limits_total_cltv_delta() {
+		let (secp_ctx, network_graph, _, _, logger) = build_graph();
+		let (_, our_id, _, nodes) = get_nodes(&secp_ctx);
+		let payee = Payee::from_node_id(nodes[6]).with_route_hints(last_hops(&nodes));
+
+		let scorer = test_utils::TestScorer::with_fixed_penalty(0);
+
+		// Make sure that generally there is at least one route available
+		let feasible_max_total_cltv_delta = 1000;
+		let route = get_route(&our_id, &payee, &network_graph, None, 100, 42, feasible_max_total_cltv_delta, Arc::clone(&logger), &scorer).unwrap();
+		let path = route.paths[0].iter().map(|hop| hop.short_channel_id).collect::<Vec<_>>();
+		assert_ne!(path.len(), 0);
+
+		// But not if we exclude all paths on the basis of their accumulated CLTV delta
+		let fail_max_total_cltv_delta = 0;
```

Comment:
Maybe set this to something non-0? Not that it matters much.

Reply:
Done in https://github.com/lightningdevkit/rust-lightning/pull/1234/commits/be78fa3e2ae094f67bf80b9ecdfc93a388b7cd5d.